sink/sink

```diff
@@ -434,6 +484,42 @@ def main():
 
     # Create the directory and chdir
     logs = os.path.expanduser(config.get("Sink", "Logs"))
+
+    # See if we have to prune logs
+    prune_interval = config.get("Sink", "PruneInterval")
+    if prune_interval >= 0:
+        status_file = os.path.expanduser(config.get("Sink", "LogsPruneStatus"))
```

Comment:
Does this need to be configurable? Can it just be a '.prune-stamp' file in the top level directory?